src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -28,60 +25,52 @@ const defaultProps = {
     },
 };
 
-class OnfidoStep extends React.Component {
-    /**
-     * @returns {boolean|*}
-     */
-    canShowOnfido() {
-        return this.props.walletOnfidoData.hasAcceptedPrivacyPolicy && !this.props.walletOnfidoData.isLoading && !this.props.walletOnfidoData.error && this.props.walletOnfidoData.sdkToken;
-    }
+function OnfidoStep({walletOnfidoData}) {
+    const {translate} = useLocalize();
+    
+    const canShowOnfido = walletOnfidoData.hasAcceptedPrivacyPolicy && !walletOnfidoData.isLoading && !walletOnfidoData.error && walletOnfidoData.sdkToken;
```

Comment:
More common naming across the project.
```suggestion
    const shouldShowOnfido = walletOnfidoData.hasAcceptedPrivacyPolicy && !walletOnfidoData.isLoading && !walletOnfidoData.error && walletOnfidoData.sdkToken;
```

Reply:
Nvm, naming it is up to you :)

Reply:
I'll actually stick with `should`, I like it!